news/_posts/2022-04-07-contributor-interview-daniel-hlavacek.md

```diff
@@ -0,0 +1,84 @@
+---
+layout: post
+title: "Merchant, freelance developer, and top contributor: Daniel Hlaváček’s experience with the PrestaShop project"
+subtitle: "Discover the faces behind the commits"
+date: 2022-04-07
+authors: [JulieVarisellaz]
+icon: icon-world
+image: /assets/images/2022/04/daniel.jpg
+tags:
+ - contribute
+ - community
+ - contribution
+---
+Have you ever wondered who was hiding behind a GitHub username? It’s time to find out! Contributor interviews are a great way of getting to know the people who participate in the PrestaShop project. This month, meet Daniel Hlaváček, aka [Hlavtox](https://github.com/Hlavtox).
+
+<img style="border: 1px solid #CCC; float: left; margin: 0 1em 1em 0;" width="517" height="388" src="/assets/images/2022/04/daniel.jpg">
+
+Daniel spends most of his time managing his family business’ online stores. The rest of the time, he works as a freelance developer for 10 clients, whose stores are all based on PrestaShop.
+
+Despite his busy schedule, Daniel also takes time to contribute to the PrestaShop open-source project, with more than [300 contributions](https://contributors.prestashop.com/). 
+
+<div style="clear:both"></div>
+
+Dear readers, meet Daniel!
+
+**Hi Daniel, can you tell us more about yourself?**
+
+Hi, I’m Daniel. I’m 28. I live in Pardubice, Czech Republic.
+
+70% of my job is to take care of my family’s online stores. The rest of the time, I work as a freelancer. I have about 10 other clients for whom I’ve built online stores. I won’t share my website here, as I’m not taking on any new clients at the moment. My hands are full! :relaxed:
+
+My family has been selling kitchen appliances online since 2006. Our main website is [Trendo.cz](https://www.trendo.cz/). My role in the family business is to take care of our stores, product catalog, network, hardware, and everything related.
+
+I have always loved computers: building and repairing laptops, making websites, setting up networks, etc. My ecommerce journey started in 2008. I was about 15 years old when I started working for my family business - managing products and content on our website. Gradually, I took on more and more responsibilities.
```

Comment:
PLS change:
`building and repairing laptops`
`building them, repairing laptops`